packages/web/src/scss/components/Container/README.md

```diff
@@ -1,13 +1,37 @@
 # Container
 
-Basic usage:
+## Basic Usage
 
 ```html
 <div class="Container">Content</div>
 ```
 
-Fluid container:
+## Fluid Container
 
 ```html
 <div class="Container Container--fluid">Content</div>
 ```
+
+## Sizes
+
+- `Container--xsmall`
+- `Container--small`
+- `Container--medium`
+- `Container--large`
+- `Container--xlarge` (default)
+
+```html
+<div class="Container Container--xsmall">Content</div>
+<div class="Container Container--small">Content</div>
+<div class="Container Container--medium">Content</div>
+<div class="Container Container--large">Content</div>
+<div class="Container Container--xlarge">Content</div>
+```
+
+⚠️ Make sure that you have properly set up `container-{{size}}-max-width` tokens in your project. Without it, the old default size will be used, and sizes will not be available.
+
```

Comment:
Could we connect the text with the list so users know it's the actual design tokens? Something like:

```suggestion

The following design tokens are required for Container to work:

```